geomstats/_backend/__init__.py

```diff
@@ -126,7 +128,9 @@
         'rand',
         'randint',
         'seed',
-        'uniform'
+        'uniform',
```

Comment:
Needs to be added in alphabetic order. This allow us to be sure that we do not add the same function twice.